less/about/preferences.less

```diff
@@ -1,12 +1,11 @@
-@import "./common.less";
```

Comment:
Moved to https://github.com/brave/browser-laptop/pull/10881/files#diff-e3eeb751016b2ce9f8278efce585a461R67 on preferences.js